Makefile

```diff
@@ -14,6 +14,10 @@ build:
 test:
 	go test -bench=. ${BUILD_DIR}/../... -v -coverprofile=coverage.out -covermode=atomic -outputdir=${BUILD_DIR}
 
+e2e-test:
+	go build -a -ldflags="-s -w -X main.version=${VERSION}" -o ${BUILD_DIR}/spot-itn ${BUILD_DIR}/../cmd/main.go
```

Comment:
curious why you are not reusing the `build` target here. Is there a specific reason?

Reply:
Yeah `make build` 's output is `-o ${BUILD_DIR}/itn-${GOOS}-${GOARCH}`, so running on mac (locally) vs. linux (gha) would have different executable names. I didn't want to share all these details with the test, so I chose a "simpler" output name for now.